admin/class-yoast-notification-center.php

```diff
@@ -96,7 +96,12 @@ public static function is_notification_dismissed( Yoast_Notification $notificati
 		$user_id       = ( ! is_null( $user_id ) ? $user_id : get_current_user_id() );
 		$dismissal_key = $notification->get_dismissal_key();
 
-		$current_value = get_user_meta( $user_id, $dismissal_key, true );
+		$current_value = get_user_option( $dismissal_key, $user_id );
+
+		// Check unprefixed user meta for backward-compatibility.
+		if ( empty( $current_value ) ) {
+			$current_value = get_user_meta( $user_id, $dismissal_key, true );
```

Comment:
If a value exists here, we should convert it to the `user_`option` and delete this one, cleaning up the database.
_We should've used prefixes for dismissal keys, to be able to do a simpler upgrade routine :(_

Reply:
Done.